seqr/views/apis/family_api.py

```diff
@@ -47,9 +47,10 @@ def family_page_data(request, family_guid):
     add_families_context(response, families, project.guid, request.user, is_analyst, has_case_review_perm)
     response['familiesByGuid'][family_guid]['detailsLoaded'] = True
 
-    outlier_samples = sample_models.filter(sample_type=Sample.SAMPLE_TYPE_RNA).exclude(rnaseqoutlier=None)
-    for sample in outlier_samples:
-        individual_guid = response['samplesByGuid'][sample.guid]['individualGuid']
+    outlier_sample_guids = sample_models.filter(sample_type=Sample.SAMPLE_TYPE_RNA)\
+        .exclude(rnaseqoutlier__isnull=True, rnaseqspliceoutlier__isnull=True).values_list('guid', flat=True)
```

Comment:
does excluding multiple conditions act as and AND or an OR

Reply:
We need to exclude `rnaseqoutlier_isnull` AND `rnaseqspliceoutlier__isnull`, which means we include `rnaseqoutlier` OR `rnaseqspliceoutlier` is not null. I can some tests for that.